libs/ledger-live-common/src/families/bitcoin/wallet-btc/xpub.ts

```diff
@@ -29,6 +29,9 @@ class Xpub {
 
   GAP = 20;
 
+  // when the difference between current blockheight and the blockheight in the storage is larger than this value, we ignore reorg
+  nbBlockIgnoreReorg = 2000;
```

Comment:
This could be set for each chain separately, with a default value, and custom values if needed.

Reply:
not relevant any more